.env.example

```
@@ -24,6 +24,10 @@ WATCH_NAMESPACE="hades"
 DELETE_ON_COMPLETE="true"
 MAX_PARALLELISM="100"
 
+### HadesLogManager Options ###
+HADESLOGMANAGER_API_PORT=8081 # Port for the Log Manager to listen on (default: 8081)
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)